core/services/pipeline/task.eth_call.go

```diff
@@ -120,7 +120,7 @@ func (t *ETHCallTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, in
 func (t *ETHCallTask) retrieveRevertReason(baseErr error, lggr logger.Logger) error {
 	reason, err := evmclient.ExtractRevertReasonFromRPCError(baseErr)
 	if err != nil {
-		lggr.Errorw("failed to extract revert reason", "baseErr", baseErr, "error", err)
+		lggr.Debugw("failed to extract revert reason", "baseErr", baseErr, "error", err)
```

Comment:
I don't have enough context either way, but did you consider `Warn`?
> - Error: Something bad happened, and it was clearly on the node op side. No need for immediate action though. Example: database write timed out
> - Warn: Something bad happened, not clear who/what is at fault. Node ops should have a rough look at these once in a while to see whether anything stands out. Example: connection to peer was closed unexpectedly. observation timed out.
> - Debug: Useful for forensic debugging, but we don't expect nops to look at this. Example: Got a message, dropped a message, ...

https://pkg.go.dev/github.com/smartcontractkit/chainlink/core/logger#Logger

Alternatively, if this is specific to mumbai, might it warrant a new chain-scoped configuration parameter?

Reply:
Yes, warn seems like a better fit thanks for the suggestion.

I am not sure if this is specific to mumbai / particular RPCs. For now warn seems like the best solution